internal/security/proxy/service.go

```diff
@@ -279,6 +399,7 @@ func (s *Service) initKongRoutes(r *KongRoute, name string) error {
 
 	switch resp.StatusCode {
 	case http.StatusOK, http.StatusCreated, http.StatusConflict:
+		s.routes[name] = r
```

Comment:
What's going on here?

Reply:
added cached routes to facilitate unit testing- a little bit of memory consumption but can increase performance and test-ability. not sure other better way to test Init() func whether additional routes has been added if not. it is internal variable